src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,73 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { selector } from '../../app/store';
+import { User } from '../../types/User';
+import { useDispatch } from 'react-redux';
+import { getUser } from '../../api';
+import { setCurrentTodo } from '../../features/currentTodo';
 
 export const TodoModal: React.FC = () => {
+  const currentTodo = selector(state => state.currentTodo);
```

Comment:
The `selector` function is used incorrectly here. It should be `useSelector` from `react-redux` to access the Redux store state. Make sure to import `useSelector` and replace `selector` with `useSelector`.